### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by truckermike** [puddletag/puddletag#763] [Q&A] [answered]
Anyone get puddletag to run in dark mode?

**Comment by audiomuze** [comment]
KDE Plasma, dark mode, puddletag follows suite.

**Reply by truckermike** [reply]
I've tried on ubuntu and manjaro both using gnome and it doesn't work on either. Flacon is the same. It also won't follow.

**Comment by JoeJoeTV** [comment, marked as answer]
How did you install puddletag?

If you use PIP, make sure, that it doesn't use the pyqt5 package installed by PIP, that wasa the problem for me. If I removed the one installed by PIP so it uses the system one, it used the same theme as other qt applications.

**Reply by truckermike** [reply]
After searching I see that pyqt5 installed with puddletag is conflicting with pyqt5-common. So I should remove the one puddletag installed?

Thank you

**Reply by JoeJoeTV** [reply]
Depends on how you installed puddletag. Did you use your distributions package manager or pip?

**Reply by truckermike** [reply]
I used the package manager.

**Reply by JoeJoeTV** [reply]
hmm, then I'm not sure, but you can still try uninstalling `pyqt5` **using pip** if you don't need it for something else. In my situation, I had puddletag installed through pip.

**Reply by truckermike** [reply]
I got it going thanks for the help.
